docs/design/runtime_processing/runtime_recording.md

```diff
@@ -348,19 +445,172 @@ end note
 deactivate APP
 
 == End recording ==
+CLI -> LTTNG : Stop & destroy LTTng session
+deactivate LTTNG
 CLI -> CARET : End message \n[/caret/end_record]
 activate CARET
 CARET -> CARET : Transition to WAIT state
 deactivate CARET
 CARET --> CLI : Send message
 deactivate CARET
-CLI -> LTTNG : Stop & destroy LTTng session
-deactivate LTTNG
 <-- CLI
 deactivate CLI
 @enduml
 ```
 
+## Sequence (multi-host system)
```

Comment:
I have some suggestions about this figure. Please look at attached image.
![image](https://github.com/tier4/caret_doc/assets/35490433/63bacc71-aac5-43cb-9ad3-a24c3e1f43f2)